.changeset/yellow-parents-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
```

Comment:
could you please split these two changesets? 
please keep description of the changeset that contains '@finos/legend-graph': patch empty